docs/customization/template.rst

```diff
@@ -98,6 +113,78 @@ Done! If you do not see any changes on the ``/admin/countries/new`` url, clear y
 
     $ php bin/console cache:clear
 
+How to customize templates via events?
+--------------------------------------
+
+Sylius uses the Events mechanism provided by the `SonataBlockBundle <https://sonata-project.org/bundles/block/master/doc/reference/events.html>`_.
+
+How to locate template events?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The events naming convention uses the routing to the place where we are adding it, but instead of ``_`` we are using ``.``,
+followed by a slot name (like ``sylius_admin_customer_show`` route results in the ``sylius.admin.customer.show.slot_name`` event).
+The slot name describes where exactly in the template's structure should the event occur.
+
+Let's see how the event is rendered in a default Sylius Admin template. This is the rendering of the event that occurs
+on the create action of Resources, at the bottom of the page (after the content of the create form):
+
+.. code-block:: twig
+
+    {# Fist we are setting the event_prefix based on route as it was mentioned before #}
+    {% set event_prefix = app.request.attributes.get('_route')|replace('_', '.') %}
```

Comment:
It has been changed to `{% set event_prefix = metadata.applicationName ~ '.admin.' ~ metadata.name ~ '.create' %}`